library/helpers/wrap.ts

```diff
@@ -32,6 +32,7 @@ export function createWrappedFunction(
   const wrapped = wrapper(original);
 
   defineProperty(wrapped, "__original", original);
+  defineProperty(wrapped, "__NR_original", original);
```

Comment:
In case the patch solution is rejected, let's do this not by default, only e.g. if some env var is set?